modules/prebidServerBidAdapter/index.js

```diff
@@ -607,7 +607,7 @@ const OPEN_RTB_PROTOCOL = {
       }
 
       if (!utils.isEmpty(videoParams)) {
-        if (videoParams.context === 'outstream' && (!videoParams.renderer || !adUnit.renderer)) {
+        if (videoParams.context === 'outstream' && (!videoParams.renderer && !adUnit.renderer)) {
```

Comment:
Maybe that double negating was causing the confusion :smile: 

What do you think of:

```suggestion
        if (videoParams.context === 'outstream' && (videoParams.renderer || adUnit.renderer)) {
```